src/scss/_draft-settings.scss

```diff
@@ -13,7 +13,7 @@
 }
 
 .draft-settings {
-  .draft-settings-duplications {
+  .draft-settings-duplications, .draft-settings-mods {
```

Comment:
This addition seems unnecessary. I think the checkbox list for the mods looks better without the additional margin.